### ISSUE TITLE
The GPU architecture in your current machine is Pascal, which is not compatible with Paddle installation with arch: 70 75 80 86 

### ISSUE THREAD

**Issue by ynes99** [PaddlePaddle/PaddleSeg#3447]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### Hello so i checked another issue with the same warning and the installation there had no problem, but mine had problems.
I did this test
`!sh tests/install/check_predict.sh` 
and all I got was blurry and pixelated pictures instead of what was supposed to be predicted.

output/result/added_prediction 
![optic_test_image](https://github.com/PaddlePaddle/PaddleSeg/assets/72978456/94c921e4-558c-4a9b-9adf-20a2c3a68d56)
 output/result/pseudo_color_prediction 
![optic_test_image](https://github.com/PaddlePaddle/PaddleSeg/assets/72978456/443c636a-90f7-4bdb-860e-8e4950e36c00)

the cuda version i'm using is 11.7 the installation is correct i checked with nvidia-smi and compiled some cuda code to be sure.
the only warning that might cause the error is this one but I installed the correct version and everything so I'm stuck on what to do

>  The GPU architecture in your current machine is Pascal, which is not compatible with Paddle installation with arch: 70 75 80 86 

the CPU installation works just fine but the GPU doesn't for some reason.

2023-08-11 13:57:19 [WARNING]	Add the `num_classes` in train_dataset and val_dataset config to model config. We suggest you manually set `num_classes` in model config.
2023-08-11 13:57:19 [INFO]	

------------Environment Information-------------
platform: Linux-5.15.0-78-generic-x86_64-with-glibc2.35
Python: 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0]
Paddle compiled with cuda: True
NVCC: Build cuda_11.7.r11.7/compiler.31442593_0
cudnn: 8.9
GPUs used: 1
CUDA_VISIBLE_DEVICES: None
GPU: ['GPU 0: NVIDIA TITAN']
GCC: gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
PaddleSeg: 2.8.0
PaddlePaddle: 2.5.1
OpenCV: 4.5.5
------------------------------------------------
2023-08-11 13:57:19 [INFO]	
---------------Config Information---------------
batch_size: 4
iters: 1000
train_dataset:
  dataset_root: data/optic_disc_seg
  mode: train
  num_classes: 2
  train_path: data/optic_disc_seg/train_list.txt
  transforms:
  - max_scale_factor: 2.0
    min_scale_factor: 0.5
    scale_step_size: 0.25
    type: ResizeStepScaling
  - crop_size:
    - 512
    - 512
    type: RandomPaddingCrop
  - type: RandomHorizontalFlip
  - brightness_range: 0.5
    contrast_range: 0.5
    saturation_range: 0.5
    type: RandomDistort
  - type: Normalize
  type: Dataset
val_dataset:
  dataset_root: data/optic_disc_seg
  mode: val
  num_classes: 2
  transforms:
  - type: Normalize
  type: Dataset
  val_path: data/optic_disc_seg/val_list.txt
optimizer:
  momentum: 0.9
  type: SGD
  weight_decay: 4.0e-05
lr_scheduler:
  end_lr: 0
  learning_rate: 0.01
  power: 0.9
  type: PolynomialDecay
loss:
  coef:
  - 1
  - 1
  - 1
  types:
  - type: CrossEntropyLoss
  - type: CrossEntropyLoss
  - type: CrossEntropyLoss
model:
  backbone:
    pretrained: https://bj.bcebos.com/paddleseg/dygraph/PP_STDCNet2.tar.gz
    type: STDC2
  num_classes: 2
  type: PPLiteSeg
------------------------------------------------

2023-08-11 13:57:19 [INFO]	Set device: gpu
2023-08-11 13:57:19 [INFO]	Use the following config to build model
model:
  backbone:
    pretrained: https://bj.bcebos.com/paddleseg/dygraph/PP_STDCNet2.tar.gz
    type: STDC2
  num_classes: 2
  type: PPLiteSeg
W0811 13:57:19.390579 55924 gpu_resources.cc:96] The GPU architecture in your current machine is Pascal, which is not compatible with Paddle installation with arch: 70 75 80 86 , it is recommended to install the corresponding wheel package according to the installation information on the official Paddle website.
W0811 13:57:19.390600 55924 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 12.2, Runtime API Version: 11.7
W0811 13:57:19.390609 55924 gpu_resources.cc:149] device: 0, cuDNN Version: 8.9.
2023-08-11 13:57:19 [INFO]	Loading pretrained model from https://bj.bcebos.com/paddleseg/dygraph/PP_STDCNet2.tar.gz
2023-08-11 13:57:20 [INFO]	There are 265/265 variables loaded into STDCNet.
2023-08-11 13:57:20 [INFO]	The number of images: 1
2023-08-11 13:57:20 [INFO]	Loading pretrained model from https://paddleseg.bj.bcebos.com/dygraph/optic_disc/pp_liteseg_optic_disc_512x512_1k/model.pdparams
2023-08-11 13:57:20 [WARNING]	ppseg_head.arm_list.0._scale is not in pretrained model
2023-08-11 13:57:20 [WARNING]	ppseg_head.arm_list.1._scale is not in pretrained model
2023-08-11 13:57:20 [WARNING]	ppseg_head.arm_list.2._scale is not in pretrained model
2023-08-11 13:57:20 [INFO]	There are 367/370 variables loaded into PPLiteSeg.
2023-08-11 13:57:20 [INFO]	Start to predict...
1/1 [==============================] - 1s 562ms/step
2023-08-11 13:57:21 [INFO]	Predicted images are saved in output/result/added_prediction and output/result/pseudo_color_prediction .

**Comment by Asthestarsfalll** [comment]
Plz report this to [Paddle](https://github.com/PaddlePaddle/Paddle) repo.
It seems that you have installed a version of CUDA that is too high.

**Comment by ToddBear** [comment]
以上回答已经充分解答了问题，如果有新的问题欢迎随时提交issue，或者在此条issue下继续回复～
我们开启了飞桨套件的ISSUE攻关活动，欢迎感兴趣的开发者参加：https://github.com/PaddlePaddle/PaddleOCR/issues/10223
